tests/Fixer/ControlStructure/TrailingCommaInMultilineFixerTest.php

```diff
@@ -28,30 +27,6 @@
  */
 final class TrailingCommaInMultilineFixerTest extends AbstractFixerTestCase
 {
-    /**
-     * @requires PHP <8.0
```

Comment:
there should be a test for <8 (ie to ensure we do not produce invalid code under <8 runtime)

Reply:
solved